### ISSUE TITLE
Additional properties

### ISSUE THREAD

**Issue by devployment** [dubek/rabbitmq-dump-queue#13]
Looks like the following properties are not included in the file

* Exchange
* RoutingKey

In my particular case they are valuable and needed.

Adding them to `func getProperties` seems to solve the issue.

I you agree putting them there, I can create a PR if wanted.

**Comment by dubek** [comment]
I assume you mean: 

> are *not* included in the file

Yes, I agree adding them to the list of properties. Please create a PR; please also add a the relevant tests in `TestFull` at the end of `main_test.go`.

**Comment by devployment** [comment]
> I assume you mean

Sure enough. Updated the org. text.

About the test. Not exactly sure how you want it to be added.

To proof the existence of  the `routing_key`, it can just be asserted as the `routing_key` will be always available through the defaults in RabbitMQ. Even if the message was not explicitly sent through an exchange with a given routing key.

To proof the additional `exchange` value, the message under test needs to be sent through an exchange. This is where multiple options arise. 

1.) We could add an additional `TestFullRouted` method doing the same as `TestFull` but on messages explicitly published through an exchange.

2.) We could sent all messages through the exchange by default and just check the existence of the `exchange` property. 

This test would not show that a message published to a queue directly will not contain the `exchange` property

What direction would you prefer?

**Comment by dubek** [comment]
I prefer your first direction (`TestFullRouted`), in which you can check for both `exchange` and `routing_key` (I hope I understand correctly, it's been a few years since I last used complex RabbitMQ topologies...).

Thanks for your contribution.

**Comment by devployment** [comment]
> I prefer your first direction (TestFullRouted)

👍 

Would you mind using an "assertion library" ?

https://github.com/stretchr/testify

It makes it a bit more readable and gives some more elaborated feedback, IMHO.

```
assert.Equal(properties["priority"], 4.0)
assert.Equal(properties["content_type"], "text/plain")
assert.Equal(properties["message_id"], "msgid-0")
```

vs

```
if properties["priority"] != 4.0 || // JSON numbers are floats
    properties["content_type"] != "text/plain" ||
    properties["message_id"] != "msgid-0" {
    t.Errorf("Wrong property value: properties = %#v", properties)
}
```

**Comment by dubek** [comment]
The code does look cleaner with testify, but for now I prefer to keep the number of dependencies to minimum, even at the cost of slightly more verbose code.

I might consider it in the future.

**Comment by flux627** [comment]
This is a needed feature for our use-case as well 👍

**Comment by devployment** [comment]
Oh. Thanks for the bump. Let me clean up things an create the PR.

**Comment by dubek** [comment]
This is now merged to master.

Soon I'll cut a new release for easier download/install for users.

**Comment by dubek** [comment]
This feature is now available in the v0.5 release of rabbitmq-dump-queue (see the [Releases](https://github.com/dubek/rabbitmq-dump-queue/releases) page).
